SWIG/options.i

```diff
@@ -791,6 +791,19 @@ class BjerksundStenslandApproximationEngine : public PricingEngine {
             const boost::shared_ptr<GeneralizedBlackScholesProcess>& process);
 };
 
+
+%{
+using QuantLib::JuQuadraticApproximationEngine;
+%}
+
+%shared_ptr(JuQuadraticApproximationEngine);
+%rename(JuQuadraticEngine) JuQuadraticApproximationEngine;
```

Comment:
I would keep the same name as the C++ class.

Reply:
Fine for me - I just kept it consitent with the `BjerksundStenslandEngine` above, but the consistency with the library is more important.